modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/AutocompleteUserMVCResourceCommand.java

```diff
@@ -105,11 +106,24 @@ private List<User> _getUsers(
 		if (CTCollectionPermission.contains(
 				permissionChecker, ctCollection, CTActionKeys.INVITE_USERS)) {
 
-			Role role = _roleLocalService.getRole(
-				themeDisplay.getCompanyId(), RoleConstants.PUBLICATIONS_USER);
+			LinkedHashMap<String, Object> params =
+				LinkedHashMapBuilder.<String, Object>put(
+					"publicationsRoleUser",
```

Comment:
How does this work? Is the `publicationsRoleUser` param defined anywhere?

If we're searching for users with specific roles, we can use the `usersRoles` param. This is defined in [UserModelPreFilterContributor](https://github.com/liferay/liferay-portal/blob/master/modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/search/spi/model/query/contributor/UserModelPreFilterContributor.java#L144-L160) and will filter the user results by the given roleIds.